ckanext/xroad_integration/commands/xroad.py

```diff
@@ -50,6 +50,16 @@ def fetch_distinct_service_stats(ctx, config, days):
     utils.fetch_distinct_service_stats(days)
 
 
+@xroad_commands.command()
+@click_config_option
+@click.pass_context
+@click.option(u'--days', type=int)
+def fetch_list_errors(ctx, config, days):
+    load_config((config or ctx.obj['config']))
```

Comment:
`click_config_option` is deprecated. Remove it, the `config` parameter and call to load_config. Click should nowadays handle all this without these.

Reply:
Updated